ocean_lib/ocean/ocean_assets.py

```diff
@@ -113,6 +123,7 @@ def create(
         owner_address: str = None,
         data_token_address: str = None,
         provider_uri=None,
+        trusted_algorithms=None,
```

Comment:
Please, do not add custom parameters to create !
If a service requires custom paramaters, add them when you create that service.
look at https://github.com/oceanprotocol/ocean.js/blob/main/test/integration/ComputeFlow.test.ts#L258

if next week we are going to add a service.type = XXX , are we going to change asset.create again?

Reply:
I think @calina-c  did this part to help me and I chose to do the `update` part. Did you find something suspicious at the error @alexcos20?

Reply:
I will take a look on create part too.

Reply:
@alexcos20 It is true, I added this part because this is what I understood from the call we had 3-4 weeks ago with you and Samer. I asked for a review on my initial changes [(the commit)](https://github.com/oceanprotocol/ocean.py/pull/139/commits/6d15e58d034af20bda400848a2af2438ea2f2d20) and whether this was the right direction to go, but I had no answer then.

So, basically, when testing, should we create the asset without the extra parameter, and apply the update_trusted_algorithms to it? Would that work?

Reply:
Alex's comment is still valid, we shouldn't add more parameters to this `assets.create`.
Instead, the trusted algorithms can be added to the attributes in the service descriptor of the compute service. 
This means that the caller has to create a computer service descriptor to be able to set trusted algorithms. Otherwise, the asset can be updated with the trusted algorithms list after publishing.